.github/workflows/review_pull_request.yml

```diff
@@ -17,7 +18,7 @@ jobs:
       - name: Checkout source code
         uses: actions/checkout@v2.3.2
 
-      - name: Load Gradle cached
+      - name: Load cached Gradle
```

Comment:
This lib is for caching, isn't it? 🤔  Should it be `Cache Gradle` instead?
Ref: https://github.com/actions/cache

Reply:
Agree, updated accordingly 🙏🏼